My Series List/My Series List/My_Series_List.xcdatamodeld/My_Series_List.xcdatamodel/contents

```diff
@@ -1,4 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="1" systemVersion="11A491" minimumToolsVersion="Automatic" sourceLanguage="Swift" usedWithCloudKit="false" userDefinedModelVersionIdentifier="">
-    <elements/>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="19206" systemVersion="20F71" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="CoreDataSeries" representedClassName="CoreDataSeries" syncable="YES" codeGenerationType="class">
```

Comment:
Can you try to rename this entity as Series?
It should be possible to have 2 Series classes in different packages.
Do you have a better idea than naming both classes (CoreData and Codable) as Series?